fleetctl/fleetctl.go

```diff
@@ -482,6 +482,36 @@ func getChecker() *ssh.HostKeyChecker {
 	return ssh.NewHostKeyChecker(keyFile)
 }
 
+func getUnitFile(file string) (*unit.UnitFile, error) {
+	var uf *unit.UnitFile
+	name := unitNameMangle(file)
+
+	log.Debugf("Looking up for Unit(%s) or its corresponding template", name)
```

Comment:
ok, I'm not completely sure about it all together but we can roll forward and clean up later if it turns out to be too noisy. One final thing though: change "Looking up for" to "Looking for" in this message. thanks